pandevice/base.py

```diff
@@ -2993,7 +2992,7 @@ def op(self, cmd=None, vsys=None, xml=False, cmd_xml=True, extra_qs=None, retry_
         """
         element = self.xapi.op(cmd, vsys, cmd_xml, extra_qs, retry_on_peer=retry_on_peer)
         if xml:
-            return ET.tostring(element)
+            return ET.tostring(element, encoding='utf-8')
```

Comment:
Nevermind, it was `encoding='unicode'` that caused the exception in python 27.  This `utf-8` solution works great.  Please disregard my comment.